src/main/target/FF_FORTINIF4/target.c

```diff
@@ -30,8 +30,17 @@
 const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
     DEF_TIM(TIM3, CH3, PB0, TIM_USE_MOTOR, 0, 0 ), // S1_OUT - DMA1_ST7
     DEF_TIM(TIM3, CH4, PB1, TIM_USE_MOTOR, 0, 0 ), // S2_OUT - DMA1_ST2
+#if defined(FF_FORTINIF4_REV03)
```

Comment:
I think we should be able to do this at runtime, with pin / timer definitions in `targetConfiguration()`. This will avoid user confusion about what version of the board they have. @jflyper can you confirm?

Reply:
@mikeller i was taking this into account but there are to many problems since lots of pins and even the use of SPI ports had been changed.

Reply:
I think SPI is also configurable in runtime by now.